contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_step_2_computer.handlebars

```diff
@@ -0,0 +1,49 @@
+<div id="clipboard" class="pull-right">
```

Comment:
Again, this file appears to be fairly redundant with what goes before - please avoid this kind of redundancy and make templates reusable.